tests/periph_uart_nonblocking/main.c

```diff
@@ -46,7 +47,7 @@ int main(void)
     _irq_disabled_print();
 
     uint32_t total_us = 0;
-    xtimer_ticks32_t counter = xtimer_now();
+    uint32_t counter = ztimer_now(ZTIMER_USEC);
```

Comment:
I'm not too familiar with ztimer, but shouldn't it be better to have `ztimer_now_t counter`? I know it resolves to the same thing currently, but I guess we want to keep the abstraction.

Reply:
> I'm not too familiar with ztimer, but shouldn't it be better to have `ztimer_now_t counter`? I know it resolves to the same thing currently, but I guess we want to keep the abstraction.

Not sure, `ztimer_now_t` made sense when we had `ztimer_now64`, now I'm not sure the abstraction makes sense to use now, and before it actually lead to casting in many places... I would rather just use `uint32_t` here.

Reply:
Ok, then maybe we will need to clarify this in the docs in some other PR, or even deprecate the type.

Reply:
For the record: I'm marking as resolved given that, currently, across the codebase we have this discrepancy and defining which version to use is out of the scope of the PR.